meow/frontend/src/components/EditPost/EditForm.js

```diff
@@ -91,7 +118,28 @@ class EditForm extends React.Component {
             <Form.Item label="twitter">
               {getFieldDecorator("post_twitter", {
                 rules: []
-              })(<TextArea rows={6} />)}
+              })(
+                <TextArea
+                  rows={6}
+                  onChange={v => {
+                    //console.log(v.target.value.length);
+                    this.setState({ twitter_length: v.target.value.normalize("NFC").length });
+                  }}
+                  maxLength={TWITTER_MAX_LENGTH}
+                />
+              )}
+              <span
+                style={{
+                  color:
+                    this.state.twitter_length < TWITTER_MAX_RECOMMENDED_LENGTH
+                      ? "black"
+                      : this.state.twitter_length > TWITTER_MAX_LENGTH
```

Comment:
Since the `TextArea` has a `maxLength` of `TWITTER_MAX_LENGTH` will this ever happen?